### ISSUE TITLE
Unable to override the default breakpoints

### ISSUE THREAD

**Issue by three60five** [aaronvanston/react-flexa#61]
I tried to set custom breakpoints using flexa object: 
```
const nightTheme = {
  flexa: {
    gutter: {
      xs: 1,
      sm: 2,
      md: 2,
      lg: 3,
    },
    breakpoints: {
      xs: 0,
      sm: 22,
      md: 44,
      lg: 66,
    },
    columns: 24,
  },
```
The columns and gutter changed but the breakpoints remained the flexa defaults. 

**Expected Behavior**
The custom breakpoints are displayed.

**Comment by angusfretwell** [comment]
Hey @three60five, have you set up a `<ThemeProvider />`? It would help to see how you're using the theme with the provider 😄

**Comment by aaronvanston** [comment]
@three60five can you provide an example of how you're using the flexa theme? For example where you set the `ThemeProvider`

**Comment by three60five** [comment]
Yes, I have set up the `<ThemeProvider>` like:
`<ThemeProvider theme={nightTheme}>`. I am using styled-components 2.1.0. As I stated before, the column and gutter overrides in the theme object are applied correctly, just not the breakpoints.

**Comment by aaronvanston** [comment]
Thanks @three60five I will take a look! Will update hopefully with a fix/solution ASAP!

**Comment by three60five** [comment]
Thanks for looking into it. If it helps, I've seeing this in a very basic app started with create-react-app where I'm checking out your library.

**Comment by aaronvanston** [comment]
@three60five To give you an update: We have located the issue at hand. When creating the media queries for the component, it does not pull from the custom theme but rather bypasses it and goes straight to the default theme. We will release a patched release ASAP. 

Thank you for spending the time to submit an issue!

**Comment by three60five** [comment]
Is there a workaround I can use today?

**Comment by aaronvanston** [comment]
The fix for the media is taking longer than expected, unfortunately, sorry!

One hacky solution, for now, is to modify the default theme inside
*/node_modules/react-felxa/dist/theme/defaultTheme.js*

We will update you when the proper solution is released.

**Comment by aaronvanston** [comment]
@three60five massive apologies for the delay on this fix. I was actually stuck on a solution for some time.

The fix for this has now been merged and release with version 0.6.0!

Thank you for spending the time to fill out an issue.
